app/views/submissions/_grades.html.erb

```diff
@@ -11,6 +11,13 @@
           <b> &ndash; </b>
         <% end %>
         <b> / <%= p.max_score ? sprintf("%.2f", p.max_score.round(2)) : raw("&ndash;") %></b>
+        <b>
+          <% if p_score&.released %>
```

Comment:
Also, it wouldn't really make sense to show this message to students. Since they see a dash if it's unreleased, and a score otherwise.

i.e. only show the message if `@cud.instructor? or @cud.course_assistant?`